code/modules/food&drinks/food/snacks/meat.dm

```diff
@@ -114,6 +114,10 @@
 	name = "synthmeat"
 	desc = "A synthetic slab of meat."
 
+/obj/item/weapon/reagent_containers/food/snacks/meat/slab/meatproduct
+	name = "synthmeat"
+	desc = "A slab of station reclaimed and chemically processed meat product."
```

Comment:
shit I forgot to change its name

we have a lot of meat slabs that are functionally identical, the value is fluff and grossout value